### PR TITLE
Added Telugu Language.

### PR DESCRIPTION
**Pull request by asaikarthikeya**
#548 

@chirag-jn Kindly review the changes.

Thank you.

### CODE REVIEW

File: src/main/res/values-bn/strings.xml [hunk 1]
```diff
@@ -43,6 +43,7 @@
     <string name="language_fr">ফ্রেঞ্চ</string>
     <string name="language_es">স্প্যানিশ</string>
     <string name="language_bn">বাংলা</string>
+    <string name="language_te">Telugu</string>
```
Comment (chirag-jn) [inline src/main/res/values-bn/strings.xml]: Shouldn't this be in the regional language too?

File: src/main/res/values-te/strings.xml [hunk 2]
```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name = "alert_delete_confirm"> మీకు ఖచ్చితంగా తెలుసా? </ string>
```
Comment (chirag-jn) [inline src/main/res/values-te/strings.xml]: Please remove extra space before and after each entry. Use refactoring in Android studio

**Review by chirag-jn — COMMENTED** [review]
Please fix merge conflicts too

**Comment by asaikarthikeya** [comment]
Resolved the issues. kindly respond quickly as hacktoberfest is closing in few hours.
@chirag-jn

**Comment by chirag-jn** [comment]
@asaikarthikeya The travis build still fails. Please check that first.

**Comment by chirag-jn** [comment]
> @asaikarthikeya The travis build still fails. Please check that first.

Fixed changes in values-te/strings.xml

File: src/main/res/values-bn/strings.xml [hunk 3]
```diff
@@ -43,6 +43,7 @@
     <string name="language_fr">ফ্রেঞ্চ</string>
     <string name="language_es">স্প্যানিশ</string>
     <string name="language_bn">বাংলা</string>
+    <string name="language_te">తెలుగు</string>
```
Comment (chirag-jn) [inline src/main/res/values-bn/strings.xml]: This doesn't match the Google Translation. Probably @Ayan-10 can help here?

**Comment by chirag-jn** [comment]
This PR is a step towards #548

**Comment by Ayan-10** [comment]
Yes. It should be written as তেলেগু instead of తెలుగు

**Review by chirag-jn — APPROVED** [review]
Seems complete now. We can merge this @cpg

**Pull request merged**